File: programmers/Lv_1/KimHyunseo/문자열다루기기본.cpp
#include <string>
#include <vector>

using namespace std;

bool solution(string s) {
	bool answer = true;
	if (s.length() == 4 || s.length() == 6)
		answer = true;
	else
		answer = false;

	for (int i = 0; i < s.length(); i++)
	{
		if (!(s[i] >= '0' && s[i] <= '9')) {
			answer = false;
			break;
		}
	}
	return answer;
}

int main()
{
	string s = "a1234";
	solution(s);
}

File: programmers/Lv_1/JeongJunKwon/문자열 다루기 기본.java
class Solution {
  public boolean solution(String s) {
      if(!(s.length() == 4 || s.length() == 6))
          return false;
      for(int i = 0; i < s.length(); i++) {
          char c1 = s.charAt(i);
          if(!('0' <= c1 && c1 <= '9'))
              return false;
      }
      return true;
  }
}